iroha/src/sumeragi.rs

```diff
@@ -53,9 +53,9 @@ pub struct Sumeragi {
     transactions_sender: TransactionSender,
     world_state_view: Arc<WorldStateView>,
     /// Hashes of the transactions that were forwarded to a leader, but not yet confirmed with a receipt.
-    transactions_awaiting_receipts: Arc<HashSet<Hash>>,
+    transactions_awaiting_receipts: Arc<DashSet<Hash>>,
```

Comment:
Maybe I missed that... Why don't we use here `HashSet`?

Reply:
DashSet is a lockfree HashSet which was hidden under `iroha_structs` crate, due to restructuring this, the `iroha_structs` crate was removed and so we have this name here, from the semantical point of view they are the same.